src/components/templates/Search/index.tsx

```diff
@@ -3,10 +3,13 @@ import { QueryResult } from '@oceanprotocol/lib/dist/node/metadatacache/Metadata
 import SearchBar from '../../molecules/SearchBar'
 import AssetQueryList from '../../organisms/AssetQueryList'
 import styles from './index.module.css'
+import PriceFilter from './filterPrice'
```

Comment:
consistenty, decide on one name and then stick to it in file, component, and import naming